### DISCUSSION TITLE
Release 5.2.0  Missing variables Datapipeline Provision

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#544] [Bugs] [answered]
Hi Team!

Below are the variables which are not there in the 5.2.0 private repo, due to this few of the DP jobs failed at the time of provisioning.
Kindly check . @santhosh-tg 
##################
1. Private repo variables:

 

####analytics spark#########
cloud_public_storage_accountname: ""
sunbird_private_storage_account_name: ""
sunbird_private_storage_region: ""

 

sunbird_public_storage_account_name: ""
sunbird_public_storage_region: ""
sunbird_public_storage_account_key: ""

 

####Druid service########
sunbird_private_azure_report_container_name: 
sunbird_public_azure_report_container_name: 
sunbird_private_s3_storage_key: ""
sunbird_druid_storage_account_name: ""
druid_azure_container_name: ""   // not yet defined

 

sunbird_druid_storage_account_key: "" 
sunbird_private_s3_storage_secret: ""

#############

Regards,
Gohila

**Comment by chitranshu-keshav** [comment]
@santhosh-tg Can you please do the needful and get this thread closed

**Reply by santhosh-tg** [reply, marked as answer]
Hi @anandp504  @manjudr @sadanandGowda It seems old variables are still referenced in the druid / spark provision. Kindly check on this and update it with the new vars. 
@gohilamariappan `cloud_public_storage_accountname` should already be part of the private template vars . Pls do check

**Reply by santhosh-tg** [reply]
Hi @anandp504  @manjudr did you get a chance to check on this issue ?

**Reply by Krishnaj20** [reply]
@gohilamariappan - Is this issue addressed? Pls confirm and mark the thread as answered if your query is addressed.

**Comment by gohilamariappan** [comment]
Its resolved.
